pyveda/vv/labelizer.py

```diff
@@ -58,6 +58,11 @@ def __init__(self, vset, mltype=None, count=None, classes=None, include_backgrou
         self.iflagged_tiles = []
         self.include_background_tiles = include_background_tiles
         self.id = []
+        self.fname = fname
+        if fname:
+            self.vb_root = store.vedabase.VedaBase.from_path(fname=self.fname)
```

Comment:
We should just import the things we want from the store and stream modules instead of importing the entire module into the namespace: `from pyveda.vedaset import VedaBase, VedaStream, ...`